variants/wifi_slot/pins_arduino.h

```diff
@@ -0,0 +1,125 @@
+/*
+  pins_arduino.h - Pin definition functions for Arduino
+  Part of Arduino - http://www.arduino.cc/
+
+  Copyright (c) 2007 David A. Mellis
+  Modified for ESP8266 WIFIO board by Ivan Grokhotkov, 2015.
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General
+  Public License along with this library; if not, write to the
+  Free Software Foundation, Inc., 59 Temple Place, Suite 330,
+  Boston, MA  02111-1307  USA
+
+  Changed : 24.11.2017 Vasily Basalaev
+            Definition for WiFi Slot
+            Amperka LLC
+            http://amperka.ru
+
+  $Id: wiring.h 249 2007-02-03 16:52:51Z mellis $
+*/
+
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        17
+#define NUM_ANALOG_INPUTS       1
+
+// TODO: this should be <= 9 if flash is in DIO mode
+#define isFlashInterfacePin(p)      ((p) >= 6 && (p) <= 11)
+
+#define analogInputToDigitalPin(p)  ((p > 0) ? NOT_A_PIN : 0)
+#define digitalPinToInterrupt(p)    (((p) < EXTERNAL_NUM_INTERRUPTS)? (p) : NOT_AN_INTERRUPT)
+#define digitalPinHasPWM(p)         (((p) < NUM_DIGITAL_PINS && !isFlashInterfacePin(p))? 1 : 0)
+
+#define PIN_SPI_SS   (15)
+#define PIN_SPI_MOSI (13)
+#define PIN_SPI_MISO (12)
+#define PIN_SPI_SCK  (14)
+
+static const uint8_t SS    = PIN_SPI_SS;
+static const uint8_t MOSI  = PIN_SPI_MOSI;
+static const uint8_t MISO  = PIN_SPI_MISO;
+static const uint8_t SCK   = PIN_SPI_SCK;
+
+#define PIN_WIRE_SDA (2)
+#define PIN_WIRE_SCL (0)
+
+static const uint8_t SDA = PIN_WIRE_SDA;
+static const uint8_t SCL = PIN_WIRE_SCL;
+
+#define PIN_A0 (3)
+#define PIN_A1 (1)
+#define PIN_A2 (16)
+#define PIN_A3 (13)
+#define PIN_A4 (5)
+#define PIN_A5 (14)
+#define PIN_A6 (4)
+#define PIN_A7 (12)
+
+static const uint8_t A0 = PIN_A0;
+static const uint8_t A1 = PIN_A1;
+static const uint8_t A2 = PIN_A2;
+static const uint8_t A3 = PIN_A3;
+static const uint8_t A4 = PIN_A4;
+static const uint8_t A5 = PIN_A5;
+static const uint8_t A6 = PIN_A6;
+static const uint8_t A7 = PIN_A7;
+
+static const uint8_t LED_BUILTIN = 2;
```

Comment:
please use a `#define LED_BUILTIN` like in the other variants and get rid of `BUILTIN_LED`, it is defined in common.h that you should use to avoid duplicate code.

Is that fair ?

Reply:
Thanks for your feedback.
I'll fix the `#define LED_BUILTIN`. Unfortunately, I can not just connect `common.h`, because the pinout of our board conflicts with this:
https://github.com/esp8266/Arduino/blob/1a9403df1b8aa06430ac7480c0a1f9729eaa987c/variants/generic/common.h#L50-L52

I could change the `common.h` in this part to this code:
```
#ifndef PIN_A0
#define PIN_A0 (17)
#endif

static const uint8_t A0 = PIN_A0;
```
but will you accept this change?

Reply:
I don't see any problem with this proposal,
as long as the API remains compatible with the Arduino use of `analogRead(A0)`
which seems to be the case in your PR and your new analog pins and I guess a analog multiplexer.